tests/integ/modin/io/test_to_pandas.py

```diff
@@ -25,3 +29,7 @@ def test_pd_to_pandas():
         check_index_type=False,
         check_dtype=False,
     )
+    assert_series_equal(
+        pd.to_pandas(pd.Series(data["c"])),
+        native_pd.Series(data["c"]),
+    )
```

Comment:
How come check_index_type=False and check_dtype=False are not needed here?

Reply:
because to_pandas support timedelta well.